Rip out libc version check.
Our glibc is newer than 2.1.1
The musl libc has an entirely unrelated version number
and of course does not support gnu_get_libc_version()

--- configure.ac	2016-01-27 01:58:25.000000000 +0100
+++ configure.ac	2016-02-23 19:21:58.906456302 +0100
@@ -9676,21 +9676,6 @@
 fi
 AC_SUBST(SYSTEM_LIBEXTTEXTCAT_DATA)
 
-dnl ***************************************
-dnl testing libc version for Linux...
-dnl ***************************************
-if test "$_os" = "Linux"; then
-    AC_MSG_CHECKING([whether libc is >= 2.1.1])
-    exec 6>/dev/null # no output
-    AC_CHECK_LIB(c, gnu_get_libc_version, HAVE_LIBC=yes; export HAVE_LIBC)
-    exec 6>&1 # output on again
-    if test "$HAVE_LIBC"; then
-        AC_MSG_RESULT([yes])
-    else
-        AC_MSG_ERROR([no, upgrade libc])
-    fi
-fi
-
 dnl =========================================
 dnl Check for the Windows  SDK.
 dnl =========================================
